app/lib/three_scale/policies/schemas/apicast-policy-v1.1.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$id": "http://apicast.io/policy-v1.1/schema",
```

Comment:
The current version of the apicast policy schema is v1.1: https://github.com/3scale/APIcast/blob/5c33a1b896545a1421dc4162ea8651f389dcaa6d/gateway/src/apicast/policy/manifest-schema.json

Added in https://github.com/3scale/APIcast/commit/9e1a5c7b1b794d134c353c98f9a853a8d90b4bd5

Reply:
Yes, copied from there and adjusted

Reply:
The problem then is we have to keep our copy synced with the original. Would it be possible to download and use the original? Is this the same as the `APICAST_REGISTRY_URL` variable? For consistency it would be good to use that variable as a single source of truth. Does this make any sense?

Reply:
`APICAST_REGISTRY_URL` doesn't expose that manifest, AFAIK.